bucket/jstock.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://jstock.org",
+    "description": "JStock is a free stock market software for 28 countries",
+    "license": "GPL-2.0-only",
+    "version": "1.0.7.45",
+    "extract_dir": "jstock",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/yccheok/jstock/releases/download/release_1-0-7-45/jstock-1.0.7.45-bin.zip",
+            "hash": "6a795a61258c3d9488b40cb449a1de9ea263624bd1a0e6c0ea73de6293e8a75b"
+        }
+    },
+    "pre_install": "Set-Content \"$dir\\jstock.bat\" '@start javaw.exe -jar \"%~dp0jstock.jar\" %*' -Encoding Ascii",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "shortcuts": [
+        [
+            "jstock.bat",
+            "JStock"
+        ]
+    ]
```

Comment:
```suggestion
    "version": "1.0.7.45",
    "description": "Stock investments tracker and manager",
    "homepage": "https://jstock.org",
    "license": "GPL-2.0-only",
    "suggest": {
        "JDK": [
            "java/oraclejdk",
            "java/openjdk"
        ]
    },
    "architecture": {
        "64bit": {
            "url": "https://github.com/yccheok/jstock/releases/download/release_1-0-7-45/jstock-1.0.7.45-bin.zip",
            "hash": "6a795a61258c3d9488b40cb449a1de9ea263624bd1a0e6c0ea73de6293e8a75b"
        }
    },
    "extract_dir": "jstock",
    "pre_install": "Set-Content \"$dir\\jstock.bat\" '@start javaw.exe -jar \"%~dp0jstock.jar\" %*' -Encoding Ascii",
    "shortcuts": [
        [
            "jstock.bat",
            "JStock"
        ]
    ],
    "checkver": {
        "github": "https://github.com/yccheok/jstock",
        "regex": "jstock-([\\d.]+)-bin\\.zip"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/yccheok/jstock/releases/download/release_$dashVersion/jstock-$version-bin.zip"
            }
        }
    }
```

Reply:
I had not thought of adding the two section as you just did. nice addition